catalyst_voices/apps/voices/lib/pages/workspace/workspace_empty_state.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:catalyst_voices/widgets/empty_state/empty_state.dart';
+import 'package:catalyst_voices_blocs/catalyst_voices_blocs.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class WorkspaceEmptyState extends StatelessWidget {
+  const WorkspaceEmptyState({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return _Selector(
+      builder: (context, state) {
```

Comment:
How about calling it after what it represents? My argument is that `state` can be anything and you need to look elsewhere to see what it is.
```suggestion
      builder: (context, showEmptyState) {
```